vuepress/docs/next/tutorials/ecr/deploy-components-without-bundle.md

```diff
@@ -55,58 +63,47 @@ If for example during setup wizard you chose a custom organization `myorg` and t
-```
 
-> **Warning**
->
-> If you change the target image of the docker build, remember to update
-> the plugin metadata in the bundle accordingly.
 
-## 2. Publish the Docker image to Docker registry (DockerHub or equivalent)
+::: tip Warning
+If you change the target image of the docker build, remember to update
+the plugin metadata in the bundle accordingly. You will also have to change the plugin custom resource when you deploy the plugin below in the `bundle/plugins` folder to point to the correct image.
+:::
 
-Let’s now publish the docker image for the microservice to make it
-available later during bundle installation in the cluster.
+## Publish the Docker image 
+
+Publish the docker image for the microservice to make it
+available for the bundle installation in the cluster. Docker Hub or an equivalent registry will do.
 
     docker push <name-of-the-image:tag>
 
-## 3. Deploy the plugin into your Entando cluster
+## Deploy the plugin 
 
  You can now deploy the plugin custom resource generated by the Entando Blueprint in the `bundle/plugins` folder.
 
-> **Warning**
->
-> As stated in step 1, if you changed the target image of your docker build, the plugin custom resource
-> in the `bundle/plugins` folder needs to be updated to point to the correct image
-
-From the JHipster project directory:
+From the JHipster project directory, deploy the plugin:
 
 ```$bash
 cd bundle/plugins
 
 kubectl create -f <plugin-file.yaml> -n entando
 ```
 
-## 4. Wait for your plugin to be in `Running` state and link the plugin with the app using an `EntandoAppPluginLink` custom resource
+## Link the plugin to the App
 
-Once the plugin server deployment is up and running, you can create an [EntandoAppPluginLink custom resource](../../docs/concepts/custom-resources.md)
+Once the plugin server deployment is up and running, create an [EntandoAppPluginLink custom resource](../../docs/concepts/custom-resources.md)
 to make the plugin API available from the EntandoApp domain.
 
-Here an example of a EntandoAppPluginLink custom resource. Some assumptions with this custom resource:
+Here is an example of a EntandoAppPluginLink custom resource. Some defaults of this custom resource are:
 - The EntandoPlugin generated with the blueprint is `my-demo-plugin`
-- The EntandoApp exposing the `my-demo-plugin` APIs is `quickstart`. This is the app name in an environment built using the Getting Started instructions.
-- Both are deployed on the `entando` namespace.
-- The link itself is named `quickstart-to-my-demo-plugin-link`
+- The EntandoApp exposing `my-demo-plugin` API is `quickstart`. This is the app name in an environment built in the Getting Started instructions.
+- Both are deployed on the `entando` namespace
+- The link is named `quickstart-to-my-demo-plugin-link`
 - The name of the link yaml is `my-link.yaml`
 
-> **Warning**
->
-> Remember to change the fields to match your setup.
+::: tip Warning
```

Comment:
Should this be an instruction? I don't know if it's referring to the snippet below or something else.

Reply:
Just a tip as we discussed